OxfordBodleian/Juel-Jensen/BDLaethd20.xml

```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                
+                                <msItem xml:id="ms_i3.2.16.1">
+                                    <locus from="54va" to="55vb"/>
+                                    <title type="complete" ref="LIT3601Miracle"/>
+                                    <textLang mainLang="gez"/>
+                                    
+                                    <incipit type="inscriptio" xml:lang="gez">
+                                        <locus target="#54va"/>
+                                        <hi rend="rubric">ተአምሪሃ፡ ለእግዝእ</hi>ትነ፡ ቅድስት፡ ድንግል፡ 
+                                        <hi rend="rubric">በ፪ማርያም፡
+                                            ወላዲ</hi>ተ፡ 
+                                    </incipit>
+                                    
+                                    <incipit type="supplication">
+                                        <locus target="#54va"/>
+                                        ጸሎታ፡ ወበረከታ፡ ወምሕረተ፡ ፍቁር፡ ወልዳ፡ የሃሉ፡ ምስለ፡ 
+                                        ገብራ<hi rend="rubric">፨</hi> 
+                                        <persName ref="PRS12911KidanaK"><hi rend="rubric">ኪዳነ፡
+                                            ክርስቶስ<supplied reason="omitted" resp="DR">፡</supplied></hi></persName>
+                                        ለዓለመ፡ ዓለም፡ አሜን<hi rend="rubric">፤</hi>
+                                    </incipit>
+                                    
+                                    <incipit>
+                                        <locus from="54va" to="54vb"/>
+                                        ወሀሎ፡ <hi rend="rubric">፩</hi> ብእሲ፡ ባዕ<cb n="b"/>ል፡ በደሴተ፡ ቌላስይስ፡ ባዕል፡ ጥቀ<hi rend="rubric">፨</hi> 
+                                        ወሖረ፡ ትርኵ፡ በሐመር፡ ከመ፡ ይጽብዖሙ፡ ሰቍዝ። ወተራከብዎ፡ በጽንፈ፡ ባሕር፡ ወነደፍዎ፡
+                                        በ<hi rend="rubric">፲</hi> ወ<hi rend="rubric">፩</hi> አሕፃ፡ 
+                                        ወእ<add place="above">ም</add>ሕፃሰ፡ <hi rend="rubric">፩</hi> እምኔሆሙ፡ ነደፎ፡ ዓይኖ፡ ወአልጸቀ፡
+                                        ለመዊት፡ ወኮነ፡ ፅቡረ<hi rend="rubric">፨</hi>  ወእምዝ፡ ይቤሎሙ፡ ለሰብኡ፡ አዕትውኒ፡ ቤትየ፡ ወአተውዎ፡ ቤቶ፡ 
+                                        ወአሕፃሰ፡ <hi rend="rubric">፫</hi> ዘውስተ፡ ዓይኑ፡ ወሥጋሁ፡ ወጽኡ፡ ወ<hi rend="rubric">፩</hi> ዘውስተ፡
+                                        ዓይኑ፡ ኢወጽአ።
+                                    </incipit>
+                                    
+                                    <explicit xml:lang="gez">
+                                        <locus from="55va" to="55vb"/>
+                                        ወገብረ፡ ምጽዋተ፡ በዙኃ፡ ለነዳያን፡ ወለምስኪናን፡ ወለቤተ፡ ክርስቲያን፡ ወኀቤሃ<hi rend="rubric">፨</hi> ዘይደሉ<hi rend="rubric">፨</hi> 
+                                        ወሶበ፡ ርእየት፡ እግዝእትነ፡ ቅድስት፡ ድንግል፡ በ<hi rend="rubric">፪</hi> <hi rend="rubric">ማርያም</hi> 
+                                        ወላዲተ፡ አምላክ፡ ቲብዓተ፡ ልቡ፡ ተመልሐ፡ ሐፅ፡ እምዓይኑ፡ ወእምዓይነ፡ ምስል፡ ዘሰምዕ፡ በአሐቲ፡ ሌሊት<hi rend="rubric">፨</hi> 
```

Comment:
The word ቲብዓተ፡ does not exist. The right expression should be ትብዓተ፡ ልቡ፡ ("with a steady heart" or something similar). It seems to me that a little dash is placed above the letter ቲ